cli-toolchain/quickstart.md

```diff
@@ -220,6 +266,37 @@ var apiBetatestGraphQLAPIEndpointOutput = process.env.API_BETATEST_GRAPHQLAPIEND
 Behind the scenes, the CLI automates populating of the resource identifiers for the selected resources as Lambda environment variables which you will see in your function code as well. This process additionally configures CRUD level IAM policies on the Lambda execution role to access these resources from the Lambda function. For instance, you might grant permissions to your Lambda function to read/write to a DynamoDB table in the Amplify project by using the above flow and the appropriate IAM policy would be set on that Lambda function's execution policy which is scoped to that table only.
 
 
+### Storage Examples
+
+#### S3 Lambda Triggers
+
+You can associate a trigger to an S3 bucket managed by the Amplify CLI, by following the `amplify add/update storage` flows. When attempting to add/update an S3 storage resource, you would get the following CLI prompts to add a trigger for it.
+
+```bash
+? Do you want to add a Lambda Trigger for your S3 Bucket? Yes
+? Select from the following options 
+❯ Choose an existing function from the project 
+  Create a new function 
+```
+As you can see in the prompt above, you can either choose to use an already existing Lambda function created using the CLI as a part of this project using `amplify add function` or create a new function with a base Lambda function to handle S3 events. We also auto-populate the IAM policies required by the Lambda execution role of the newly created function to access the S3 bucket.
+
+***Note***: You can associate only one Lambda Function trigger to an S3 bucket.
+
+#### DynamoDB Lambda Triggers
+
+You can associate a trigger to a DynamoDB table  managed by the Amplify CLI, by following the `amplify add/update storage` flows. When attempting to add/update a DynamoDB storage resource, you would get the following CLI prompts to add a trigger for it.
```

Comment:
1) extra space between "table" and "managed"
2) You can associate a Lambda trigger with a DynamoDB table, managed by the Amplify CLI, using the `amplify add/update storage` flows.